#goal definition
int32 base_request
---
#result definition
int32 base_result
---
#feedback
int32 base_feedback
